content/post/prioritization_meeting.md

```diff
@@ -0,0 +1,60 @@
+useful agile processes that our team follows.
+
+### Sprint prioritization basics
+
+The key focus areas of our scrum process are: sprint planning, daily stand ups, backlog grooming, end of sprint retro and sprint prioritization. The prioritization meeting occurs at the end of each sprint, is facilitated by either a scrum master or Data lead, and includes representatives of each function that Data supports. That is, if a function in the org has work that they would like the Data team to work on in the upcoming sprint, that function would send a representative to the meeting.
+Others are welcome to sit in or dial in, but at minimum some stakeholder representation is required.
+
+The purpose of the meeting is essentially for stakeholders to work with Data to come up with a rank-ordered list of tickets for consideration in Data's next sprint. The entire group must come to a consensus around which ticket should be ranked first, second, etc... The meeting facilitator literally moves tickets up and down the board and the group decides (debates?) where a ticket should land. This way the Data team's workload is aligned with the organizations priorities without Data having to chase down stakeholders and decide what is the most important intitiative this sprint. If Marketing and Product disagree on what is most important, the entire room together can weigh in and in the end one of the tickets is ranked above the other. When prioritization works well, it is a thing of beauty for some of the following reasons.
+
+### Setting stakeholder expectations
+
+As mentioned, with everyone in the room representatives of each function that attend the meeting get insight into what the Data team is working on. As a result stakeholders are less likely to be confused as to why their work is not being prioritized. This reduces possible resentment or the potential for slacking analysts with extraneous requests. The prioritization meeting establishes a two-way communication between Data and the rest of the org. We acknowledge that we understand business priorities and will put a good faith effort to working on it. And if we don't include a ticket in the sprint, the onus is on us to explain why. We are all held accountable.
+
+### Workload transparency
+
+If Marketing concedes that Product's A/B test analysis is the most important thing Data can work on this sprint, then when they ask why we don't have time to work on their attribution model and we point to the A/B test, the conversation is much easier since there is a built in understanding. Better yet, Marketing can look on our board (assuming project management is visible to the org) and see what we are working on. This transparency means less chance for misunderstanding around Data team priorities.
```

Comment:
What if neither team concedes their "want" is lower priority than the other? Does Data play a role in helped them find a ranking? Any advice for how to play mediator?